docs/UserGuide.adoc

```diff
@@ -49,7 +49,7 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 == Adding a person: `add`
 
-Adds a person to the address book. +
+Adds a person to the address book in alphabetical order. +
```

Comment:
```suggestion
Adds a person to the address book. +
```

It may be preferable to leave it as is, so that the user's understanding is that the address book is always sorted (as the list command implies, and maybe you would want to clarify somewhere else as well). In any case, there is no option to add a person in "not alphabetical order".